src/data/gravity.graphql

```diff
@@ -1055,9 +1055,11 @@ type LotEvent {
   createdAt: String!
   internalID: ID!
   key: String!
+  paddleNumber: String
   type: String!
   userAgent: String
   userID: ID
+  userName: String
```

Comment:
I don't get why this change is included 🤔

Reply:
just because someone else made updates to the Gravity schema and didn't pull them in to Metaphysics (I think Ohm may be referencing GravQL directly)